server/routes/createGoal/createGoalView.ts

```diff
@@ -0,0 +1,22 @@
+import type { CreateGoalForm } from 'forms'
+import type { PrisonerSummary } from 'viewModels'
+
+export default class CreateGoalView {
+  constructor(
+    private readonly prisonerSummary: PrisonerSummary,
+    private readonly createGoalForm: CreateGoalForm,
+    private readonly errors?: Array<Record<string, string>>,
+  ) {}
+
+  get renderArgs(): {
+    prisonerSummary: PrisonerSummary
+    form: CreateGoalForm
+    errors?: Array<Record<string, string>>
+  } {
+    return {
+      prisonerSummary: this.prisonerSummary,
+      form: this.createGoalForm,
+      errors: this.errors || [],
+    }
+  }
```

Comment:
These three view objects are identical right? I don't follow why we can't use the same one, or perhaps use inheritance 🤔

Reply:
I had the same thought TBH .... lets talk it through